pkg/reconciler/kubernetes/tektoninstallerset/install.go

```diff
@@ -71,15 +71,36 @@ type installer struct {
 	Manifest mf.Manifest
 }
 
-func (i *installer) EnsureCRDs() error {
-	if err := i.Manifest.Filter(mf.Any(mf.CRDs)).Apply(); err != nil {
+func ensureResources(mani *mf.Manifest) error {
```

Comment:
will this allow to update resources if it is changed?
for eg. when user changes some fields in TektonConfig for config default/feature flag
we update the configmap

Reply:
no, that is tricky.
either we can make all updates immutable.
or move the allResourcesExists Check to after the Apply() call. this second option is not perfect but it could be good enough

Reply:
updated the logic 🧑‍💻